src/SFA.DAS.FindEpao.Web/Extensions/DateFormatExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace SFA.DAS.FindEpao.Web.Extensions
+{
+    public static class DateFormatExtensions
+    {
+        public static string LongDateFormat(this DateTime? sourceDateTime)
```

Comment:
Do you have any tests around the new format extensions?

Reply:
tests added

Reply:
Could we pull apart the tests for the Date and String version extensions into their own files. I think the date test should also define the expected outcome in the test case, because at the moment it is repeating the logic it is trying to test.

Reply:
split files for extension methods (Date and String). Also added explicit check for date